src/XR/motionController/webXRAbstractMotionController.ts

```diff
@@ -291,7 +291,10 @@ export abstract class WebXRAbstractMotionController implements IDisposable {
          * handedness (left/right/none) of this controller
          */
         public handedness: MotionControllerHandedness,
-        _doNotLoadControllerMesh: boolean = false
+        /**
+         * @hidden
+         */
+        public _doNotLoadControllerMesh: boolean = false

```

Comment:
>_doNotLoadControllerMesh [](http://example.com/codeflow?start=15&length=24)

if this is public should it be renamed?

Reply:
It should be a hidden public member, with the @hidden tag on it, though I've never seen it used inline in this way. @RaananW just double-checking, this would work as expected and hide the member from the public, right?

Reply:
yes, this is "internal public". Underscore means - don't use outside of the framework. There is mainly because there is really no private and public in JS (in ES5 to be exact)